ipi/inputs/motion/instanton.py

```diff
@@ -131,6 +131,39 @@ class InputInst(InputDictionary):
                 "help": "Allows to specified non uniform time discretization as proposed in J. Chem. Phys. 134, 184107 (2011)",
             },
         ),
+        "friction": (
```

Comment:
In this keyword, as well as frictionSD and fric_spec_dens, I thinkt he help string needs more information. For example in the Laplace transform, better give the reference and eq. for definition in the paper. Other keywords could also cite paper for equations and what it means.

Reply:
sure